docs/ml-features.md

```diff
@@ -37,7 +37,7 @@ In the following code segment, we start with a set of sentences.  We split each
 Refer to the [HashingTF Scala docs](api/scala/index.html#org.apache.spark.ml.feature.HashingTF) and
 the [IDF Scala docs](api/scala/index.html#org.apache.spark.ml.feature.IDF) for more details on the API.
 
-{% include_example scala/org/apache/spark/examples/ml/TfIdfExample.scala %}
```

Comment:
There is no need to change `TfIdfExample` to `HashingTF`. You can treat examples in the section [`Feature Extractors`](http://spark.apache.org/docs/latest/ml-features.html#feature-extractors) as examples, since `TF-IDF`, `Word2Vec`, and `CountVectorizer` have already been written with `include_example`.